doc/userguide/src/SupportingTools/Libdoc.rst

```diff
@@ -199,20 +206,24 @@ If needed, the format can be explicitly set with the :option:`--format` option.
    python -m robot.libdoc OperatingSystem OperatingSystem.xml
    python -m robot.libdoc test/resource.robot doc/resource.libspec
    python -m robot.libdoc --format xml MyLibrary MyLibrary.spec
-   python -m robot.libdoc --format xml:html MyLibrary MyLibrary.xml
+   python -m robot.libdoc --format xml -s html MyLibrary MyLibrary.xml
```

Comment:
For consistency reasons it would be better to use either long or short options in the last example.